R/onLoad.R

```diff
@@ -0,0 +1,12 @@
+.onAttach <- function(libname, pkgname) {
+  # switch the default ggplot2 theme to theme_dreal
```

Comment:
Ca sent la reprise de code de `drealdown` :smile: 

Sérieusement, s'il y a eu beaucoup de code de repris, il faut absolument vérifier la compatibilité des licences et créditer les détenteurs de droits de propriété intellectuelle.

Reply:
tout à fait, il faut créditer. La licence est MIT pour drealthemes.y a deux fonctions :  check_font_in_R et load_font qui sont reprises

Reply:
OK, j'ai vu que [le cph était ThinkR](https://gitlab.com/dreal-datalab/drealdown/-/blob/c7bdbad17d65ed517bc89ef33e1030ef8bb68e59/LICENSE.md#L3).

Reply:
Je demande à seb comment il veut qu'on procède

Reply:
oh, pas besoin de demander : la licence est MIT avec cph ThinkR. Il suffit de les rajouter dans le fichier `DESCRIPTION`.

Reply:
solved in https://github.com/spyrales/gouvdown/pull/15/commits/6ac22b572b277493dc39e2e9279e971fe634a45c